packages/ui-components/src/components/Inputs/Autocomplete.tsx

```diff
@@ -44,9 +44,6 @@ const StyledAutocomplete = styled(MuiAutocomplete)`
       border-color: ${props => props.theme.palette.error.main};
     }
   }
-  .MuiFormControl-root {
-    margin-bottom: 0;
-  }
```

Comment:
@tcaiger I removed this since it was causing the input fields to be too bunched up.

Not sure how this will affect datatrak, perhaps something to do with the error messaging?

Reply:
I think this is fine, we can just add in any extra styling to datatrak-web instead. This was my bad